README.md

```diff
@@ -34,3 +34,38 @@ dataset = app.create_dataset(dataset_id="demo_dataset")
 # execute data upload to Clarifai app dataset
```

Comment:
just noticing in the docstring at the top we just say Export your PAT as an environment variable. Let's be sure in all these kind of comments to say CLARIFAI_PAT env variable so we're helping the user at every opportunity where we say those things.

Reply:
Certainly. Will ensure that the PAT requirement is incorporated wherever feasible.